templates/index.html

```diff
@@ -2,14 +2,30 @@
 
 <h1 class="title">{{title}}</h1>
 
+<table>
+    <thead>
+        <tr>
+            <th><center>Organization Types&nbsp</center></th>
+            <th><center>&nbspNumber of Organizations</center></th>
```

Comment:
Let's not use &nbsp and I think Bulma has some class to center a table text and we don't need to use the <center> tag. Please check.